blueprints/codes/eurocode/nen_en_1993_1_1_c2_a1_2016/chapter_6_ultimate_limit_state/formula_6_5.py

```diff
@@ -0,0 +1,44 @@
+"""Formula 6.5 from NEN-EN 1993-1-1+C2+A1:2016: Chapter 6 - Ultimate limit state."""
+# pylint: disable=arguments-differ
+from blueprints.codes.eurocode.nen_en_1993_1_1_c2_a1_2016 import NEN_EN_1993_1_1_C2_A1_2016
+from blueprints.codes.formula import Formula
+from blueprints.type_alias import KN
+
+
+class Form6Dot5UnityCheckTensileStrength(Formula):
+    """Class representing formula 6.5 for the unity check for tensile strength."""
+
+    label = "6.5"
+    source_document = NEN_EN_1993_1_1_C2_A1_2016
```

Comment:
Also it struck me that this is not an exact representation of the EC formula as it is an in-equality there --> I think we should discuss that also (Note: I don't have a problem with the current implemenation because it clearly is a unity check and not a check if the unity-check is correct)

Reply:
We discussed today that the ratio form is preferable. Can you resolve based on the discussion?